rbac/management/group/definer.py

```diff
@@ -31,7 +31,8 @@ def seed_group(tenant):
     """For a tenant create or update default group."""
     with tenant_context(tenant):
         name = 'Default user access'
-        group_description = 'Default group that contains default permissions for new users.'
+        group_description = 'This group contains the roles that all users inherit by default.\
```

Comment:
I believe this will add some unnecessary whitespace to the saved string. Maybe something like this?
```python
group_description = ("This group contains the roles that all users inherit by default. "
                    "Adding or removing roles in this group will affect permissions for all users in your organization.")
```

Reply:
Thanks, it is fiexed now :)